GitUI/TaskbarProgress.cs

```diff
@@ -7,7 +7,7 @@ public static class TaskbarProgress
     {
         private static void Try(Action<TaskbarManager> action)
         {
-            if (GitCommands.Utils.EnvUtils.RunningOnWindows() && TaskbarManager.IsPlatformSupported)
+            if (GitCommands.Utils.EnvUtils.RunningOnWindowsWithMainWindow() && TaskbarManager.IsPlatformSupported)
```

Comment:
Nit: Not yours but noticed.
```suggestion
            if (EnvUtils.RunningOnWindowsWithMainWindow() && TaskbarManager.IsPlatformSupported)
```

Reply:
Fixed